### PR TITLE
Add unstarted status

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Makes `blockchain_status`  be `UNSTARTED` when a transfer is `resolve_as_rejected`'d  
image

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by michielderoos** [comment]
[Blockchain status rejected/unstarted if approval is rejected](https://trello.com/c/aC2xw4Ab/647-blockchain-status-rejected-unstarted-if-approval-is-rejected)

File: app/migrations/versions/bb92df9cb6e3_.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# revision identifiers, used by Alembic.
+revision = 'bb92df9cb6e3'
+down_revision = '2f8c71039480'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("ALTER TYPE blockchainstatus ADD VALUE 'UNSTARTED'")
```
Comment (tristanhcole) [inline app/migrations/versions/bb92df9cb6e3_.py:21]: I think we should probably also convert the previous Credit Transfer's `blockchain_status=UNSTARTED` whom have `transfer_status=REJECTED`

File: app/migrations/seed.py [hunk 2]
```diff
@@ -388,6 +390,15 @@ def create_float_transfer_account(app):
...
     print_section_conclusion('Done Creating/Updating Float Wallet')
 
+def make_rejected_transactions_unstarted(app):
+    print_section_title('Marking rejected transactions as unstarted')
+    rejected_transactions = db.session.query(CreditTransfer)\
+        .filter(CreditTransfer.transfer_status == TransferStatusEnum.REJECTED)\
+        .execution_options(show_all=True)
+    for rt in rejected_transactions:
+        rt.blockchain_status = BlockchainStatus.UNSTARTED
+    db.session.commit()
```
Comment (tristanhcole) [inline app/migrations/seed.py:400]: @michielderoos can we please have this as a raw SQL operation directly in the `bb92df9cb6e3_.py` migration. This means the data is changed only once when DB is migrated and in a sequential order that is expected. As `seed.py` is run basically every time the app is spun up or updated

**Review by tristanhcole — CHANGES_REQUESTED** [review]
See https://github.com/teamsempo/SempoBlockchain/pull/503/files#r511713520

File: app/migrations/seed.py (hunk 2 above)
Comment (tristanhcole) [reply app/migrations/seed.py:400]: I also spoke to @enjeyw about this and he concurs

**Comment by michielderoos** [comment]
> See https://github.com/teamsempo/SempoBlockchain/pull/503/files#r511713520

@tristanhcole Makes sense, Done!

**Review by tristanhcole — APPROVED** [review]
Tested and works well for me!

**Pull request merged**